HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/URLOpener.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  URLOpener.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+public struct URLOpener {
+    private let application: URLOpenerProtocol
+
+    public init(_ application: URLOpenerProtocol) {
+        self.application = application
+    }
+
+    func openWebsite(url: URL, completion: ((Bool) -> Void)?) {
```

Comment:
[Suggest]: maybe we can think about renaming `openWebsite` -> openLink ? What do you think?